Tests/BaseObjectTests.swift

```diff
@@ -16,181 +18,183 @@ class BaseObjectTests: QuickSpec {
-                    let anotherObject = BaseObject()
+                        expect(callbackWasCalled) == false
+                    }
 
-                    baseObject.on(eventName, callback: callback)
+                    it("doesn't executes callback for another context object") {
+                        let anotherBaseObject = ConcreteBaseObject()
 
-                    anotherObject.trigger(eventName)
+                        baseObject.on(eventName, callback: callback)
+                        anotherBaseObject.trigger(eventName)
 
-                    expect(callbackWasCalled) == false
+                        expect(callbackWasCalled) == false
+                    }
                 }
-            }
 
-            describe("once") {
-                it("Callback should be called on event trigger") {
-                    baseObject.once(eventName, callback: callback)
-                    baseObject.trigger(eventName)
+                describe("once") {
+                    it("executes callback function") {
+                        baseObject.once(eventName, callback: callback)
+                        baseObject.trigger(eventName)
 
-                    expect(callbackWasCalled) == true
-                }
+                        expect(callbackWasCalled) == true
+                    }
 
-                it("Callback should not be called twice") {
-                    baseObject.once(eventName, callback: callback)
+                    it("doesn't execute callback function twice") {
+                        baseObject.once(eventName, callback: callback)
 
-                    baseObject.trigger(eventName)
-                    callbackWasCalled = false
-                    baseObject.trigger(eventName)
+                        baseObject.trigger(eventName)
+                        callbackWasCalled = false
+                        baseObject.trigger(eventName)
 
-                    expect(callbackWasCalled) == false
-                }
+                        expect(callbackWasCalled) == false
+                    }
 
-                it("Callback should not be called if removed") {
-                    let listenId = baseObject.once(eventName, callback: callback)
-                    baseObject.off(listenId)
-                    baseObject.trigger(eventName)
+                    it("doesn't execute callback function when it is removed") {
+                        let listenId = baseObject.once(eventName, callback: callback)
+                        baseObject.off(listenId)
+                        baseObject.trigger(eventName)
 
-                    expect(callbackWasCalled) == false
+                        expect(callbackWasCalled) == false
+                    }
                 }
-            }
 
-            describe("listenTo") {
-                it("Should fire callback for an event on a given context object") {
-                    let contextObject = BaseObject()
+                describe("listenTo") {
```

Comment:
According to the Project conventions, we should add the # prefix on the Method Name.

``context("#listenTo")``